optimizely/config/datafileprojectconfig/mappers/experiment.go

```diff
@@ -65,9 +65,10 @@ func mapExperiment(rawExperiment datafileEntities.Experiment) entities.Experimen
 	experiment := entities.Experiment{
 		AudienceIds:           rawExperiment.AudienceIds,
 		ID:                    rawExperiment.ID,
+		LayerID:               rawExperiment.LayerID,
```

Comment:
This doesn't seem to be used. Can you remove it?

Reply:
done!